src/libraries/AndGPT.UI.Core/src/Services/EventLogService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Diagnostics;
+
+namespace AndGPT.UI.Core.Services;
+
+// EventLog is mostly windows, so we put it in the UI.Core, which is a de facto windows container right now.
+
+public class EventLogService
```

Comment:
I intended to use this, but didn't.   We should expand on this as a diagnostic service